error.md

```diff
@@ -0,0 +1,62 @@
+# Errors faced during migration with nikita v1
+
+## Installation of semver necessary
+
+When running the `bootstrap` command, I got an error stating that `semver` module was not found.
+
+### Solution
+
+npm install semver
+
+## SSH not working but no error
+
+Nikita actions are not being executed. The code exits after entering the input
+
+### Solution
+
+Worked after refactoring the code in `lib/run.coffee`
+
+```coffee
+  nikita
+    .log.cli(config.cli_config)
+    .log.md(config.log_path)
+    .ssh.open(metadata: header: 'SSH open', config.ssh)
+    .ssh.close(metadata: header: 'SSH closed')
+```
+
+## SSH connection not established
+
+Connection cannot be established.
+
+### Solution
+
+The issue was due to improper config object property. The config object passed was the following:
+
+```coffee
+{
+  disabled: false,
+  host: '192.168.1.171',
+  port: '22',
+  password: 'rcarcarca'
+}
+```
+
+I noticed several things.
+
+* First, connection `port` should be an integer to work.
+* Second, `disabled` has no effect here and should be passed in `nikita.ssh.open`
+
+Using this config makes things working.
+
+```coffee
+  config_test = {
```

Comment:
no idea how `config_test` can be used in the code

Reply:
It was illustrative to say that the `port` needed to be an integer which was not the case before.

I am going to specify it in the document.

Reply:
this is not the case, string are coerced to integer, are you sure you are linking nikita ? it doesn't look like because this was implemented after alpha 0